repos/system_upgrade/el7toel8/actors/redhatsignedrpmcheck/tests/test_redhatsignedrpmcheck.py

```diff
@@ -30,7 +30,7 @@ def __call__(self, **report_fields):
 
 
 def test_skip_check(monkeypatch):
-    monkeypatch.setattr(os, "getenv", lambda(_): True)
+    monkeypatch.setattr(os, "getenv", lambda(_): True)  # noqa: E999
```

Comment:
actually using _ is forbidden in leapp, due to localization strings.
```suggestion
    monkeypatch.setattr(os, "getenv", lambda(_unused): True)  # noqa: E999
```

Reply:
Thanks. I completely forgot about that. Is it mentioned somewhere? Guidelines, conding standards, etc.?

Reply:
Not sure, it might have been forgotten to add